R/AffineScoreFunction.R

```diff
@@ -107,10 +107,10 @@ setMethod("*", signature("numeric", "AffineUnconditionalScore"),
 
 
 
-setClass("AffineConditionalScore", contains = c("AffineScore", "AbstractConditionalScore", "ConditionalScore"))
```

Comment:
whats happening here?

Reply:
Sorry, I forgot to explain this. I think this was not correct, because a ConditionalScore needs a slot "distribution". Analogously, an AffineUnconditionalScore is also just an UnconditionalScore and not an IntegralScore.

Reply:
Just type 2 * ConditionalPower(Normal(), PointMassPrior(0, 1)) and you'll see what I mean.

Reply:
makes sense